main.js

```diff
@@ -147,6 +155,7 @@ async function initDatabases() {
       table.string('universe');
       table.json('setArguments');
       table.integer("fadeTime").defaultTo(null);
+      table.integer('folderId').defaultTo(1);
```

Comment:
Ah so for legacy databases this is a breaking change?

Reply:
Yes, but that's the case with most of the updates to this project and just something we accepted :)